ledger/src/blocktree_processor.rs

```diff
@@ -565,15 +552,13 @@ fn process_pending_slots(
         }
 
         trace!(
-            "Bank for {}slot {} is complete. {} bytes allocated",
+            "Bank for {} slot {} is complete. {} bytes allocated",
```

Comment:
oops, the lack of space here is intentional.  Otherwise you get two spaces: "root..slot"

Reply:
oh i see, thanks!